lib/bedrock.js

```diff
@@ -645,6 +698,57 @@ function _startWorker(state, script) {
     };
     worker.send({type: 'bedrock.runOnce', id: msg.id, done: false});
   });
+
+  worker.on('message', function(msg) {
+    if(!_isMessageType(msg, 'bedrock.runOnceAsync')) {
```

Comment:
Looks like this is the same as the `bedrock.runOnce` handler?  Could the message type and handler be reused in the async function?